packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -87,14 +88,28 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
 
   @override
   Widget build(BuildContext context) {
+    final Widget summaryTree = Expanded(
```

Comment:
remove the types. Unlike the package:flutter code base we don't need the extra unneeded types.
So
`final Widget summaryTree` to `final summaryTree`

Reply:
Yes the habits of writing the package:flutter code just carried on, sorry for that :joy: will fix it!